tests/sunpath_test.py

```diff
@@ -0,0 +1,68 @@
+import math
+import pytest
+from pathlib import Path
+from ladybug.epw import EPW
+from ladybug.location import Location
+from ladybug.sunpath import Sunpath, Point3D, Vector3D
+from ladybug_vtk.to_vtk import create_polyline, create_points
+from ladybug_vtk.model_dataset import ModelDataSet
+from ladybug_vtk.model import Model
+
+
+def test_sunpath(temp_folder):
```

Comment:
This is a great example. We can write this as a function and inject it to the ladybug Sunpath class. I can show you some sample code.

Reply:
Here is the module that extends honeybee and adds radiance properties to it:
https://github.com/ladybug-tools/honeybee-radiance/blob/master/honeybee_radiance/_extend_honeybee.py

The module is then called from inside `__init__.py`. We can work on that after merging this PR.

Reply:
I sent a PR to address this comment: https://github.com/devngc/ladybug-vtk/pull/2